impls/monero.rs/src/lib.rs

```diff
@@ -0,0 +1,260 @@
+use std::ffi::{CStr, CString};
+use std::os::raw::{c_char, c_int, c_void};
+use std::path::PathBuf;
+use std::ptr::NonNull;
+use std::sync::Arc;
+
+use libloading::{Library, Symbol};
+
+#[cfg(target_os = "android")]
+const LIB_NAME: &str = "libmonero_libwallet2_api_c.so";
+#[cfg(target_os = "ios")]
+const LIB_NAME: &str = "MoneroWallet.framework/MoneroWallet";
+#[cfg(target_os = "linux")]
+const LIB_NAME: &str = "monero_libwallet2_api_c.so";
+#[cfg(target_os = "macos")]
+const LIB_NAME: &str = "monero_libwallet2_api_c.dylib";
+#[cfg(target_os = "windows")]
+const LIB_NAME: &str = "monero_libwallet2_api_c.dll";
+
+pub mod network {
```

Comment:
You can use the const values generated exposed by `bindings.rs` here:
```rust
pub const NetworkType_MAINNET: ::std::os::raw::c_int = 0;
pub const NetworkType_TESTNET: ::std::os::raw::c_int = 1;
pub const NetworkType_STAGENET: ::std::os::raw::c_int = 2;
```

Reply:
Resolved by 78b1aef99f5a37ac86ecb1031a265cb7a090d1bb